pkgs/development/libraries/libspatialite/default.nix

```diff
@@ -31,11 +31,15 @@ stdenv.mkDerivation rec {
     geos # for geos-config
   ];
 
+  # From: https://www.gaia-gis.it/fossil/libspatialite/info/7c452740fe
+  # see also: https://github.com/NixOS/nixpkgs/issues/347085
+  patches = [ ./xmlNanoHTTPCleanup.patch ];
```

Comment:
It would be nice to move the comment inside so that it clearly matches the specific patch if more patches are added:

```nix
  patches = [
    # Drop use of deprecated libxml2 HTTP API.
    # From: https://www.gaia-gis.it/fossil/libspatialite/info/7c452740fe
    # see also: https://github.com/NixOS/nixpkgs/issues/347085
    ./xmlNanoHTTPCleanup.patch
  ];
```

Also, patches conventionally belong just after `src`.

Reply:
> Also, patches conventionally belong just after src.

Indeed, it's better! Do you know if there is a document somewhere describing these kinds of convention or is it just a usage thing?

Reply:
I collected some of them here https://github.com/jtojnar/nixpkgs-hammering but mostly you need to get feel for conventions by osmosis.